backend/src/main/java/saci/application/model/ScoreRequest.java

```diff
@@ -0,0 +1,45 @@
+package saci.application.model;
+
+import java.util.List;
+
+public class ScoreRequest {
+    private Long roleId;
+    private List<Answer> answers;
+
+    public Long getRoleId() {
+        return roleId;
+    }
+
+    public void setRoleId(Long roleId) {
+        this.roleId = roleId;
+    }
+
+    public List<Answer> getAnswers() {
+        return answers;
+    }
+
+    public void setAnswers(List<Answer> answers) {
+        this.answers = answers;
+    }
+
+    public static class Answer {
```

Comment:
Even though ScoreRequest depends on Answer, the Answer class should hav it's own file.